pureskillgg_dsdk/tome/manifest.py

```diff
@@ -79,17 +109,22 @@ def content_name(page_number, subtype):
     return f"{subtype}_{str(page_number).zfill(5)}"
 
 
-def create_manifest(tome_name, path, ds_type, is_header, header_tome_name, src_id):
+def create_manifest(
+    tome_name, ds_type, is_header, header_tome_name, src_id, is_copied_header=False
+):
+    key = "/".join(
+        ["tome", ds_type, tome_name, "header" if is_copied_header else "", "tome"]
```

Comment:
Same comment about joining an empty string. Just want to make sure we don't miss any.

Reply:
👍